misc/scripts/install-local.sh

```diff
@@ -92,9 +92,13 @@ function loggingMeta() {
 
 
 function aria2 {
+Parallel=$nproc + 5
```

Comment:
I think it's best to use `bc` for safe string operation

Reply:
Codefactor has a better option

Reply:
May you edit this?? I'm not familiar with bc

Reply:
Codefactor had a better option, actually

Reply:
Ok, did changes